src/endpoints/transactions/entities/ppu.metadata.ts

```diff
@@ -0,0 +1,25 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class PpuMetadata {
+    @ApiProperty({
+        description: 'Last processed block number',
+        example: 47428477,
+    })
+    LastBlock: number = 0;
```

Comment:
lowercase, maybe? `lastBlock` instead of `LastBlock`, `fast` instead of `Fast`...

Reply:
done